### ISSUE TITLE
False Positive in Windows third-party antivirus

### ISSUE THREAD

**Issue by tomodachi94** [gorilla-devs/GDLauncher#736]
**Describe the bug**
I got a virus alert for `GDLauncher.exe` from Avast Antivirus.

**To Reproduce**
1. Download and install GDLauncher.
2. Download and install Avast Antivirus.
3. Run GDLauncher with antivirus active.
4. Create a Twitch instance with Direwolf20 1.12.
5. Wait a minute or two.
6. See icon for GDLauncher disappear and "Virus found" popup from Avast pop up.

**Expected behavior**
I expected the program to run normally and without Avast detecting a virus.

**Screenshots**
![Virus popup](https://user-images.githubusercontent.com/68489118/103432489-724faf80-4b94-11eb-9f4d-aed09b3075a2.png)

**Operating System:**
 - OS: Windows 10
 - Version: 10.0.18363 Build 18363

**Comment by Ladvace** [comment]
This happen because we don't a certificate (actually we have one but it doesn't work) that tells the os we are a safe app.

**Comment by Ladvace** [comment]
just whitelist gdlauncher in your antivirus (in this case avast)

**Comment by tomodachi94** [comment]
> just whitelist gdlauncher in your antivirus (in this case avast)

That was not the point of the issue. It was to notify them that I received a false-positive, not ask for help @Ladvace.

**Comment by Ladvace** [comment]
Yeah I'm sorry didn't want to seems rude, the point was that we are working on that and it's not and "easy fix"

**Comment by tomodachi94** [comment]
Most things I find aren't an "easy fix," I understand this.
